dask/dataframe/dispatch.py

```diff
@@ -83,7 +85,33 @@ def tolist(obj):
     return func(obj)
 
 
-def make_meta_util(x, index=None, parent_meta=None):
+def make_meta(x, index=None, parent_meta=pd.DataFrame()):
```

Comment:
Can we make `parent_meta=None` by default and handle updating `parent_meta` to `pd.DataFrame()` inside `make_meta` to avoid having a mutable default for a keyword argument? For example, we could handle defaulting to the pandas backend in the `else` block below

Reply:
Sure, thanks @jrbourbeau for the catch. I have made this change.